haystack/nodes/reader/table.py

```diff
@@ -252,3 +253,203 @@ def _calculate_answer_offsets(answer_coordinates: List[Tuple[int, int]], table:
 
     def predict_batch(self, query_doc_list: List[dict], top_k: Optional[int] = None, batch_size: Optional[int] = None):
         raise NotImplementedError("Batch prediction not yet available in TableReader.")
+
+
+class RCIReader(BaseReader):
+    """
+    Table Reader model based on Glass et al. (2021)'s Row-Column-Intersection model.
+    See the original paper for more details:
+    Glass, Michael, et al. (2021): "Capturing Row and Column Semantics in Transformer Based Question Answering over Tables"
+    (https://aclanthology.org/2021.naacl-main.96/)
+
+    Each row and each column is given a score with regard to the query by two separate models. The score of each cell
+    is then calculated as the sum of the corresponding row score and column score. Accordingly, the predicted answer is
+    the cell with the highest score.
+    """
```

Comment:
Can we add one line that explains the core pros/cons to TableReader? (e.g. confidence scores, aggregation options ...)